docs/operating/operator-introduction.md

```diff
@@ -10,27 +10,29 @@ comment:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-*This guide is for people who want to register with Neon EVM as a proxy operator for processing transactions on Solana. We assume you have prior experience compiling Solana-based blockchain nodes, or have worked as a blockchain validator.*
+*This guide is for people who want to register with Neon EVM as a Proxy Operator for processing transactions on Solana. We assume you have prior experience compiling Solana-based blockchain nodes, or have worked as a blockchain validator.*
 
 ## Introduction
-Ethereum users may run their smart contract-based applications on Solana via Neon EVM. Modified smart contracts generate transactions in accordance with the Ethereum rules. These are submitted them to the Neon EVM and broadcast as Neon transactions. Since the structures of Ethereum and Solana transactions differ from each other, Neon EVM contains special proxy servers that are responsible for converting and executing transactions in Solana. These servers are configured and maintained by Neon EVM Proxy Operators.
+Ethereum dApps may run their smart contract-based applications on Solana via Neon EVM. Modified smart contracts generate transactions in accordance with the Ethereum rules. These are submitted to the Neon EVM and broadcast as Neon transactions. Since the structures of Ethereum and Solana transactions differ from each other, Neon EVM contains special Proxy servers that are responsible for converting and executing transactions in Solana. These servers are configured and maintained by Neon EVM Proxy Operators.
 
-This guide contains a list of the Proxy Operator's duties, as well as instructions to ensure the successful operation of a proxy and the successful execution of transactions.
+This page introduces the Proxy Operator's duties, and the following pages provide instructions to ensure the successful operation of a Proxy; including the successful execution of transactions.
 
 ## Duties of a Proxy Operator
-A Proxy Operator's main task is to install software on a server in order to accept a transaction formed according to Ethereum rules, and to ensure its execution in Solana. This involves:
+A Proxy Operator's main task is to install software on a server in order to provide an RPC endpoint to accept transactions formed according to Ethereum rules, and ensures their execution in Solana. 
+
+This involves:
```

Comment:
@afalaleev I am not convinced that this full list is applicable--- I find it hard to imagine that the Operator has anything to do with line 27 "Convert txs to Solana format" -- that must be handled by the Proxy ?? Please comment!

Reply:
Yep, you are right
Operator runs Neon Proxy, which packs (converts) Neon transactions into Solana transactions